### DISCUSSION TITLE
Unable to convert SVG to TIFF with ccittfax4 compression and resolution unit 2

### DISCUSSION THREAD

**Discussion by nishantjiit** [libvips/libvips#2625] [Q&A]
I am unable to convert SVG to TIFF with ccittfax4 compression and resolution unit 2. Either of compression/resolution unit is applied at once.

What I want:

SVG to TIFF
Compression: CCITT T.6
Resolution Unit: 2 (Inch)
Horizontal/Vertical Resolution: 300DPI

`vips.exe extract_band ../../input.svg ../../output.tiff[squash,compression=ccittfax4,strip] 1`

The above line converts SVG to TIFF with compression CCITT T.6.

`vips.exe tiffsave ../../output.tiff ../../final-output.tiff --resunit=inch --xres=11.8`

The above line applies resolution unit =2 & horizontal/vertical resolution = 300 DPI but now the compression is gone. It says compression = Uncompressed.

**Comment by jcupitt** [comment]
Hi @nishantjiit,

Try:

```
vips extract_band ../../input.svg ../../output.tiff[squash,compression=ccittfax4,bitdepth=1] 1
vips copy ../../input.svg ../../output.tiff[squash,compression=ccittfax4,bitdepth=1,resunit=inch,xres=11.8]
```

`squash` is deprecated, use `bitdepth`. You can put any `--flag` argument in the list of save options.
